.gitignore

```diff
@@ -2,8 +2,38 @@ packwiz-installer-bootstrap.jar
 packwiz-installer.jar
 completion.sh
 reload.bat
-# Exclude exported CurseForge zip files
-*.zip
+config/sodium*
+config/iris*
+.fabric
+backups
+coremods
+crash-reports
+ESM
+fancymenu*
+!fancymenustuff
+item_renderer
+local
+logs
+saves
+screenshots
+ModTranslations
+observable_profiles
+rei_exports
+shaderpacks
+shaderpacks/*
+texturepacks
+hs_err_pid*
+o_prof
+imgui.log
+ears_debug.log
+hotbar.nbt
+rhino.local.properties
+servers.dat*
+usercache.json
+journeymap
+!journeymap/defaultconfig.json
+mods
+servers.dat*
```

Comment:
Same as the mods folder mentioned before, adding the servers.dat here to the gitignore will prevent future modifications to the default server list.